src/components/form/Form.jsx

```diff
@@ -98,6 +180,539 @@ const Form = () => {
           />
         </div>
       );
+    case 30:
+      return (
+        <div>
+          <Question19a
+            nextStep={nextStep}
+            prevStep={prevStep}
+            handleChangeTest={handleChangeTest}
+            values={values}
+            step={step}
+            handleSubmit={handleSubmit}
+          />
+        </div>
+      );
+    case 31:
```

Comment:
Shouldnn't this whole `switch(step)` be replaced by `<Form>`'s conditional rendering ?